### ISSUE TITLE
9.4.1.1 - changes in the ticket are not recorded

### ISSUE THREAD

**Issue by mauriciolamego** [glpi-project/glpi#5716]
**Describe the bug**

when clicking on a ticket, on the ticket tab there will be a delay of about 30 seconds to open the full view. And sometimes it does not save the changes requested on the ticket.

**Page(s) URL**

https://.../HelpDesk/front/ticket.form.php?id=xxxx

**To reproduce**

1. Go to 'https://.../HelpDesk/front/ticket.php'
2. Click on 'https://.../HelpDesk/front/ticket.form.php?id=1820'
3. Scroll down to 'change category' or other item, when I change it an erro occured, but not always;

**Expected behavior**

sometimes I request the change of category or even other items of the ticket the glpi informs that it made the change, but it effectively did not work (did not record the change). When this occurs the SQL error is shown.

**Logs**

```
Extract from .../HelpDesk/files/_log/sql-errors.log

[2019-03-31 09:27:10] glpisqllog.ERROR: DBmysql::query() in /home/storage/a/e3/16/solemtic1/public_html/HelpDesk/inc/dbmysql.class.php line 177
  *** MySQL query error:
  SQL: INSERT INTO `glpi_tickets_users` (`tickets_id`, `type`, `users_id`, `alternative_email`) VALUES ('1820', '2', '2', '')
  Error: Duplicate entry '1820-2-2-' for key 'unicity'
  Backtrace :
  inc/dbmysql.class.php:821                          
  inc/commondbtm.class.php:658                       DBmysql->insert()
  inc/commondbtm.class.php:1140                      CommonDBTM->addToDB()
  inc/commonitilobject.class.php:1828                CommonDBTM->add()
  inc/commonitilobject.class.php:883                 CommonITILObject->addAdditionalActors()
  inc/ticket.class.php:1306                          CommonITILObject->prepareInputForUpdate()
  inc/commondbtm.class.php:1362                      Ticket->prepareInputForUpdate()
  front/ticket.form.php:72                           CommonDBTM->update()
  {"user":"2@hm8263"} 

[2019-03-31 21:10:28] glpisqllog.ERROR: DBmysql::query() in /home/storage/a/e3/16/solemtic1/public_html/HelpDesk/inc/dbmysql.class.php line 177
  *** MySQL query error:
  SQL: SELECT * FROM `glpi_users` WHERE `glpi_users`.`id` = '2' LIMIT 1
  Error: MySQL server has gone away
  Backtrace :
  inc/dbmysqliterator.class.php:95                   
  inc/dbmysql.class.php:569                          DBmysqlIterator->execute()
  inc/commondbtm.class.php:297                       DBmysql->request()
  inc/session.class.php:97                           CommonDBTM->getFromDB()
  inc/auth.class.php:865                             Session::init()
  front/login.php:84                                 Auth->login()
  {"user":"@hm8263","mem_usage":"0.022\", 9.26Mio)"} 

```

**Screenshots**

**Your GLPI setup (you can find it in Setup > General menu, System tab)**

```

Information about system installation & configuration
--
[code]   GLPI 9.4.1.1 (/HelpDesk => /home/storage/a/e3/16/solemtic1/public_html/HelpDesk) Installation mode: TARBALL

Operating system: Linux hm8263 4.4.165-grsec-1.x86_64 #1 SMP Wed Nov 28 21:23:35 UTC 2018 x86_64 PHP 7.1.13 cgi-fcgi (Core, PDO, PDO_Firebird, PDO_ODBC, Phar, Reflection, SPL, SimpleXML, bcmath, bz2, calendar, cgi-fcgi, 	ctype, curl, date, dba, dom, enchant, exif, fileinfo, filter, ftp, gd, gettext, gmp, hash, iconv, imap, interbase, intl, json, 	ldap, libxml, mbstring, mcrypt, mysqli, odbc, openssl, pcntl, pcre, pdo_dblib, pdo_mysql, pdo_pgsql, pdo_sqlite, pgsql, posix, 	pspell, readline, recode, session, shmop, snmp, soap, sockets, standard, sysvmsg, sysvsem, sysvshm, tidy, timezonedb, tokenizer, 	wddx, xml, xmlreader, xmlrpc, xmlwriter, xsl, zip, zlib) Setup: max_execution_time="60" memory_limit="512M" post_max_size="50M" safe_mode="" session.save_handler="files" 	upload_max_filesize="50M"  Software: Apache () 	Mozilla/5.0 (Macintosh; Intel Mac OS X 10_14_4) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/12.1 Safari/605.1.15 Server Software: Percona Server (GPL), Release 81.0, Revision c96c427 	Server Version: 5.6.35-81.0-log 	Server SQL Mode:  	Parameters: glpi_p@glpi_p.mysql.dbaas.com.br/glpi_p 	Host info: glpi_p.mysql.dbaas.com.br via TCP/IP 	 mysqli extension is installed ctype extension is installed fileinfo extension is installed json extension is installed mbstring extension is installed iconv extension is installed zlib extension is installed curl extension is installed gd extension is installed simplexml extension is installed xml extension is installed ldap extension is installed imap extension is installed xmlrpc extension is installed exif extension is installed Zend OPcache extension is not present APCu extension is not present CAS extension is not present Database version seems correct (5.6.35) - Perfect! /home/storage/a/e3/16/solemtic1/public_html/HelpDesk/files/_log : OK /home/storage/a/e3/16/solemtic1/public_html/HelpDesk/config : OK /home/storage/a/e3/16/solemtic1/public_html/HelpDesk/files : OK /home/storage/a/e3/16/solemtic1/public_html/HelpDesk/files/_dumps : OK /home/storage/a/e3/16/solemtic1/public_html/HelpDesk/files/_sessions : OK /home/storage/a/e3/16/solemtic1/public_html/HelpDesk/files/_cron : OK /home/storage/a/e3/16/solemtic1/public_html/HelpDesk/files/_graphs : OK /home/storage/a/e3/16/solemtic1/public_html/HelpDesk/files/_lock : OK /home/storage/a/e3/16/solemtic1/public_html/HelpDesk/files/_plugins : OK /home/storage/a/e3/16/solemtic1/public_html/HelpDesk/files/_tmp : OK /home/storage/a/e3/16/solemtic1/public_html/HelpDesk/files/_cache : OK /home/storage/a/e3/16/solemtic1/public_html/HelpDesk/files/_rss : OK /home/storage/a/e3/16/solemtic1/public_html/HelpDesk/files/_uploads : OK /home/storage/a/e3/16/solemtic1/public_html/HelpDesk/files/_pictures : OK Web access to the files directory should not be allowed but this cannot be checked automatically on this instance. Make sure acces to error log file is forbidden; otherwise review .htaccess file and web server configuration.SELinux mode is Disabled

htmLawed version 1.2.4 in (/home/storage/a/e3/16/solemtic1/public_html/HelpDesk/lib/htmlawed) phpmailer/phpmailer version 6.0.7 in (/home/storage/a/e3/16/solemtic1/public_html/HelpDesk/vendor/phpmailer/phpmailer/src) simplepie/simplepie version 1.5.2 in (/home/storage/a/e3/16/solemtic1/public_html/HelpDesk/vendor/simplepie/simplepie/library) tecnickcom/tcpdf version 6.2.26 in (/home/storage/a/e3/16/solemtic1/public_html/HelpDesk/vendor/tecnickcom/tcpdf) michelf/php-markdown in (/home/storage/a/e3/16/solemtic1/public_html/HelpDesk/vendor/michelf/php-markdown/Michelf) true/punycode in (/home/storage/a/e3/16/solemtic1/public_html/HelpDesk/vendor/true/punycode/src) iamcal/lib_autolink in (/home/storage/a/e3/16/solemtic1/public_html/HelpDesk/vendor/iamcal/lib_autolink) sabre/vobject in (/home/storage/a/e3/16/solemtic1/public_html/HelpDesk/vendor/sabre/vobject/lib) zendframework/zend-cache in (/home/storage/a/e3/16/solemtic1/public_html/HelpDesk/vendor/zendframework/zend-cache/src) zendframework/zend-i18n in (/home/storage/a/e3/16/solemtic1/public_html/HelpDesk/vendor/zendframework/zend-i18n/src) zendframework/zend-serializer in (/home/storage/a/e3/16/solemtic1/public_html/HelpDesk/vendor/zendframework/zend-serializer/src) monolog/monolog in (/home/storage/a/e3/16/solemtic1/public_html/HelpDesk/vendor/monolog/monolog/src/Monolog) sebastian/diff in (/home/storage/a/e3/16/solemtic1/public_html/HelpDesk/vendor/sebastian/diff/src) elvanto/litemoji in (/home/storage/a/e3/16/solemtic1/public_html/HelpDesk/vendor/elvanto/litemoji/src) symfony/console in (/home/storage/a/e3/16/solemtic1/public_html/HelpDesk/vendor/symfony/console) leafo/scssphp in (/home/storage/a/e3/16/solemtic1/public_html/HelpDesk/vendor/leafo/scssphp/src)

```

**Additional context**

**Comment by cedric-anne** [comment]
Hi,

We made some performances fixes recently, especially 1d477e6ccfb7c6582d4c50dcfc103f05f2e2000a and f7c15e3326ed7f784f2a1327ae5d06991c16a36e.

Can you try if this problem is still present after applying these fixes ?

Regards

**Comment by mauriciolamego** [comment]
Hello Anne, thank you very much for the tip. Worked perfectly.
Helped me a lot.
